#! /bin/sh /usr/share/dpatch/dpatch-run
## fix-index-php.dpatch by Diego Iastrubni <diego.iastrubni@xorcom.com>
##
## DP: The default index.php shiped with freepbx contains broken html,
## DP: because it does not include "footer.php". This patch fixed it, 
## DP: and also unsets $_SESSION["AMP_user"], this way the "inyourface"
## DP: bar is not displayed on that page

@DPATCH@
diff -urNad freepbx-2.1.3.dfsg/amp_conf/htdocs/admin/index.php /tmp/dpep.lbyARy/freepbx-2.1.3.dfsg/amp_conf/htdocs/admin/index.php
--- freepbx-2.1.3.dfsg/amp_conf/htdocs/admin/index.php	2006-05-12 01:34:09.000000000 +0300
+++ /tmp/dpep.lbyARy/freepbx-2.1.3.dfsg/amp_conf/htdocs/admin/index.php	2006-12-06 14:35:42.000000000 +0200
@@ -94,7 +94,6 @@
 
 
 
-</div>
 
 <script language="javascript">
 <!--
@@ -107,3 +106,7 @@
 //-->
 </script>
 
+<?php
+$_SESSION["AMP_user"] = null;
+include "footer.php";
+?>
